inc/tokens/class-authorization-code.php

```diff
@@ -129,7 +159,15 @@ public function validate( $args = [] ) {
 			);
 		}
 
-		return true;
+		$code_verifier = $this->validate_code_verifier( [
+				'code_verifier' => $args['code_verifier'],
+			]
+		);
+		if ( is_wp_error( $code_verifier ) ) {
+			return $code_verifier;
+		}
+
+		return $code_verifier;
```

Comment:
The `if` isn't necessary, as we're returning the value of `$code_verifier` in both cases.

Reply:
I would rather just return ```true``` at the bottom and keep if ```if```, if there are any validations to add later it's easier. Is that fine?